### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RyanMersmann**
# Pull Request Template

## Description
Add a summary that address the following:
 - motivation/purpose for the feature 
 Update on readme for next cohort

### CODE REVIEW

**Review by charlene-johnson — APPROVED** [review]
Looks Good!

File: README.md [hunk 1]
```diff
@@ -150,11 +150,19 @@ cd story-squad-fe-b/
...
-npm install
+## Setting up the App
+Front End:
+- npm install
+- add a new .env file with the info provided by your TPL
+- npm start
+- -open localhost:3000
+Back End
+- npm install
+- add a new .env file with the info provided by your TPL (if you are using something other than Docker (like Postgres), add it to your backend .env file instead of the docker url
```
Comment (leachtucker) [inline README.md:161]: I think you may have meant ElephantSQL here. (Not ‘PostgreSQL’)

**Review by leachtucker — APPROVED** [review]
Nice job!

**Review by StefxM — APPROVED** [review]
Soooo much better! Good work

**Pull request merged**